OloEditor/SandboxProject/Assets/Scripts/LuaScripts/LuaTerrainController.lua

```
@@ -0,0 +1,47 @@
+    if terrain == nil then
+        Log.Warn("[LuaTerrainController] entity has no TerrainComponent")
+        return
+    end
+    terrain.proceduralEnabled = true
+    terrain.octaves = octaves
+    terrain.ridgeBlend = ridgeBlend
+    terrain.heightScale = heightScale
+    reseed(id)
+end
+
+function TerrainController.OnUpdate(id, dt)
```

Luacheck (1.2.0)

[warning] unused argument 'dt'

(W212)